dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/ClusterDrawable.java

```diff
@@ -1,22 +1,51 @@
 package nl.tudelft.dnainator.javafx.drawables.strains;
 
-import javafx.collections.MapChangeListener;
-import javafx.scene.Group;
-import javafx.scene.shape.Circle;
-import javafx.scene.text.Text;
 import nl.tudelft.dnainator.core.SequenceNode;
 import nl.tudelft.dnainator.core.impl.Cluster;
 import nl.tudelft.dnainator.javafx.ColorServer;
 import nl.tudelft.dnainator.javafx.drawables.Drawable;
+import nl.tudelft.dnainator.javafx.views.AbstractView;
+import nl.tudelft.dnainator.javafx.widgets.PropertyType;
+import nl.tudelft.dnainator.javafx.widgets.Propertyable;
 
+import java.util.LinkedHashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import javafx.collections.MapChangeListener;
+import javafx.scene.Group;
+import javafx.scene.shape.Circle;
+import javafx.scene.text.Text;
+
+/**
+ * This enum represents all properties a Cluster can have.
+ */
+enum ClusterPropertyTypes implements PropertyType {
+	TITLE("Cluster"),
+	ID("NodeID"),
+	SEQUENCE("Sequence"),
+	STARTREF("Startref"),
```

Comment:
Grammarnazi! "Start reference" and "End reference"? It's displayed in to the user, after all.